BoxOffice/Model/MovieDetail.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  MovieDetail.swift
+//  BoxOffice
+//
+//  Created by 이원빈 on 2023/01/02.
+//
+
+import Foundation
+
+struct MovieDetail {
+    let currentRank: String
+    let title: String
+    let openData: String
+    let totalAudience: String
+    let rankInten: String
```

Comment:
```suggestion
struct MovieDetail {
    let currentRank: String
    let title: String
    let openDate: String
    let totalAudience: String
    let rankInten: String
```

`openDate`가 의미가 더 확실할 것 같네요.
`rankInten`의 inten은 어떤걸까요..? 풀네임으로 써주거나 `rankChange`처럼 쉬운 의미도 괜찮을 것 같습니다!

Reply:
휴먼에러가 났군요 openDate가 맞네요! 순위증감분이 response에서 오는 축약형 단어를 그대로 썼네요. 이 두 부분은 수정하겠습니다